listenbrainz/db/dump_manager.py

```diff
@@ -70,6 +71,7 @@ def create(location, threads):
     create_path(dump_path)
     db_dump.dump_postgres_db(dump_path, time_now, threads)
     ls.dump_listens(dump_path, time_now, threads)
+    write_hashes(dump_path)
```

Comment:
There is no error handling here. If the disk runs out of space during hash creation, the hash creation fails silently.